.prettierrc

```diff
@@ -1,7 +1,26 @@
 {
-	"useTabs": true,
 	"tabWidth": 4,
+	"useTabs": true,
+	"printWidth": 80,
 	"semi": true,
 	"singleQuote": false,
-	"printWidth": 80
+	"plugins": ["@ianvs/prettier-plugin-sort-imports"],
```

Comment:
Question: is there a particular reason you chose a plugin for Prettier rather than something like [eslint-plugin-import](https://github.com/import-js/eslint-plugin-import) which is included in [eslint-config-react-app](https://github.com/facebook/create-react-app/tree/main/packages/eslint-config-react-app)?